content/messaging/customer-chat-api/v3.2/rtm-reference/index.mdx

```diff
@@ -868,13 +867,13 @@ A component of the [Filled form](#filled-form) event.
 | Field                   | Required | Data type               |  Notes                                                                                                   |
 | ------------------------| -------- | ----------------------- | -------------------------------------------------------------------------------------------------------: |
 | `fields`                | yes      | `array of objects`      | The fields a form contains.                                                                              |
```

Comment:
I think this one should be removed because it might cauase confusion why it's on the same level as other fields.